lib/Benchmark/Executor/template/time.template

```diff
@@ -14,6 +14,32 @@ $bootstrap = '{{ bootstrap }}';
 $parameters = {{ parameters }};
 $warmup = {{ warmup }};
 
+if (function_exists('hrtime')) {
+    // hrtime() generates the following format: [S, NS]
+    $timeFunc = 'hrtime';
+    $timeDiffFunc = function($startTime, $endTime) {
+        list($startSec, $startNano) = $startTime;
+        list($endSec, $endNano)     = $endTime;
+        $diffSec  = $endSec - $startSec;
+        $diffNano = $endNano - $startNano;
+        return ($diffSec * 1000000) + ($diffNano / 1000);
+    };
+} else {
+    // microtime() generates the following format: "0.MMMMMM00 SSSSSSSSSSS"
+    $timeFunc = 'microtime';
```

Comment:
Sorry if I sound ignorant, but why do you change this? Do we lose precision with `microtime(true) * 1E6` ?

Reply:
changed - should be fine at this point but math operations could be improved on calculating statistics.